src/nft/components/explore/CollectionTable.tsx

```diff
@@ -1,82 +1,133 @@
-import { CellProps, Column } from 'react-table'
+import { BigNumber } from '@ethersproject/bignumber'
+import { useMemo } from 'react'
+import { CellProps, Column, Row } from 'react-table'
 
 import { CollectionTableColumn } from '../../types'
-import {
-  CollectionTitleCell,
-  CommaWithDayChange,
-  EthWithDayChange,
-  WeiWithDayChange,
-  WithCommaCell,
-} from './Cells/Cells'
+import { ChangeCell, CollectionTitleCell, DiscreteNumberCell, EthCell, VolumeCell } from './Cells/Cells'
 import { Table } from './Table'
 
 export enum ColumnHeaders {
   Volume = 'Volume',
+  VolumeChange = 'Volume change',
   Floor = 'Floor',
+  FloorChange = 'Floor change',
   Sales = 'Sales',
   Items = 'Items',
   Owners = 'Owners',
 }
 
-const columns: Column<CollectionTableColumn>[] = [
-  {
-    Header: 'Collection',
-    accessor: 'collection',
-    Cell: CollectionTitleCell,
-  },
-  {
-    id: ColumnHeaders.Volume,
-    Header: ColumnHeaders.Volume,
-    accessor: ({ volume }) => volume.value,
-    sortDescFirst: true,
-    Cell: function EthDayChanget(cell: CellProps<CollectionTableColumn>) {
-      return <EthWithDayChange value={cell.row.original.volume} />
-    },
-  },
-  {
-    id: ColumnHeaders.Floor,
-    Header: ColumnHeaders.Floor,
-    accessor: ({ floor }) => floor.value,
-    sortDescFirst: true,
-    Cell: function weiDayChange(cell: CellProps<CollectionTableColumn>) {
-      return <WeiWithDayChange value={cell.row.original.floor} />
-    },
-  },
-  {
-    id: ColumnHeaders.Sales,
-    Header: ColumnHeaders.Sales,
-    accessor: 'sales',
-    sortDescFirst: true,
-    Cell: function withCommaCell(cell: CellProps<CollectionTableColumn>) {
-      return <WithCommaCell value={{ value: cell.row.original.sales }} />
-    },
-  },
-  {
-    id: ColumnHeaders.Items,
-    Header: ColumnHeaders.Items,
-    accessor: 'totalSupply',
-    sortDescFirst: true,
-    Cell: function withCommaCell(cell: CellProps<CollectionTableColumn>) {
-      return <WithCommaCell value={{ value: cell.row.original.totalSupply }} />
-    },
-  },
-  {
-    Header: ColumnHeaders.Owners,
-    accessor: ({ owners }) => owners.value,
-    sortDescFirst: true,
-    Cell: function commaDayChange(cell: CellProps<CollectionTableColumn>) {
-      return <CommaWithDayChange value={cell.row.original.owners} />
-    },
-  },
-]
-
 const CollectionTable = ({ data }: { data: CollectionTableColumn[] }) => {
+  const floorSort = useMemo(() => {
+    return (rowA: Row<CollectionTableColumn>, rowB: Row<CollectionTableColumn>) => {
+      const aFloor = BigNumber.from(rowA.original.floor.value)
+      const bFloor = BigNumber.from(rowB.original.floor.value)
+
+      return aFloor.gte(bFloor) ? 1 : -1
+    }
+  }, [])
```

Comment:
does this mean you only do this calculation once and don't update on prop changes?

Reply:
yes - you technically don't need to memo it the react-docs just rec that you do and like this